.github/workflows/test_and_build.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.7, 3.8]
+        python-version: ["3.9", "3.10", "3.11"]
```

Comment:
The lowest version of python you allow for the package is still 3.7, so I think we should still test as low as that

Reply:
Flake8 requires python >3.8.1. I think I might as well drop 3.7 at least.